R/oauth-flow-auth-code.R

```diff
@@ -59,7 +59,7 @@ req_oauth_auth_code <- function(req, client,
                                 pkce = TRUE,
                                 auth_params = list(),
                                 token_params = list(),
-                                redirect_uri = "http://localhost",
+                                redirect_uri = Sys.getenv("HTTR2_OAUTH_REDIRECT_URL", "http://localhost"),
```

Comment:
I'd suggest making this an exported helper function so you can document how the default is computed.

Reply:
Like `default_redirect_uri()`?

Reply:
Yeah, exactly.

Reply:
Done.